plugins/modules/packaging/language/pear.py

```diff
@@ -202,7 +265,8 @@ def main():
         argument_spec=dict(
             name=dict(aliases=['pkg'], required=True),
             state=dict(default='present', choices=['present', 'installed', "latest", 'absent', 'removed']),
-            executable=dict(default=None, required=False, type='path')),
+            executable=dict(default=None, required=False, type='path'),
+            prompts=dict(default=None, required=False, type='list', elements='raw')),
```

Comment:
```suggestion
            prompts=dict(default=None, required=False, type='list', elements='raw'),
        ),
```
Then the next addition's diff will only add one line and not touch this line as well.